<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Учет сотрудников организаций</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_currentOrg">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Нет организации</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_departments">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="b_add">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>-31072</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Добавить сотрудника</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/resources/icons/add_employee.ico</normaloff>:/resources/icons/add_employee.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_delete">
        <property name="text">
         <string>Удалить сотрудника</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/resources/icons/remove_employee.ico</normaloff>:/resources/icons/remove_employee.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_previewSave">
        <property name="text">
         <string>Сохранить</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/resources/icons/save.ico</normaloff>:/resources/icons/save.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_previewSaveAs">
        <property name="text">
         <string>Сохранить как...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/resources/icons/save_as.ico</normaloff>:/resources/icons/save_as.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_submitChanges">
        <property name="text">
         <string>Принять изменения</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/resources/icons/accept.ico</normaloff>:/resources/icons/accept.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_revertChanges">
        <property name="text">
         <string>Отменить изменения</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/resources/icons/cancel.ico</normaloff>:/resources/icons/cancel.ico</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>Правка</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_saveAs"/>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_selectNewDatabase"/>
    <addaction name="separator"/>
    <addaction name="action_sendToServer"/>
    <addaction name="action_receiveFromServer"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_sendToServer">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/upload.ico</normaloff>:/resources/icons/upload.ico</iconset>
   </property>
   <property name="text">
    <string>Отправить базу данных на сервер</string>
   </property>
  </action>
  <action name="action_receiveFromServer">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/download.ico</normaloff>:/resources/icons/download.ico</iconset>
   </property>
   <property name="text">
    <string>Получить базу данных с сервера...</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_saveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/save.ico</normaloff>:/resources/icons/save.ico</iconset>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
   <property name="toolTip">
    <string>Сохранить базу данных как</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/open.ico</normaloff>:/resources/icons/open.ico</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
   <property name="toolTip">
    <string>Открыть существующий файл базы данных</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_selectNewDatabase">
   <property name="text">
    <string>Выбрать другую базу данных...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
